### ISSUE TITLE
Use-cases section 404

### ISSUE THREAD

**Issue by shawnthompson** [w3c/wcag3#224]
The [Use Cases for WCAG 3.0 Conformance](https://w3c.github.io/silver/use-cases/) is giving a 404.

Could it be because of the [auto-publish.yml](https://github.com/w3c/silver/blob/main/.github/workflows/auto-publish.yml) action is missing it in the [matrix](https://github.com/w3c/silver/blob/main/.github/workflows/auto-publish.yml#L14) section?
